engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,11 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
```

Comment:
@PramodSSImmaneni I agree with @sgolovko. Not all functionality related to log4j and logging that we support for applications should be supported by the apex client. For example, Apex supports changing logger levels in containers for running applications and Apex client log levels are set during the start of the apex client. The same applies to dt.attr.DEBUG attribute. It is an application level setting that does not apply to the apex client, AFAIK.